scripts/gha/build_desktop.py

```diff
@@ -264,6 +277,7 @@ def parse_cmdline_args():
   parser.add_argument('--config', default='Release', help='Release/Debug config')
   parser.add_argument('--target', nargs='+', help='A list of CMake build targets (eg: firebase_app firebase_auth)')
   parser.add_argument('--target_format', default=None, help='(Mac only) whether to output frameworks (default) or libraries.')
+  parser.add_argument('--use_openssl', action='store_true', default=None, help='Use openssl for build instead of boringssl')
```

Comment:
What do you thin about just adding an openssl flag to the prereqs script also? (And we can move the Windows openssl setup into there too.)

Reply:
Thats a good point but as per our original design, we wanted to keep all architecture specific steps under build_desktop. And if we are using vcpkg, we have to specify the exact architecture to build. Something to discuss in our "various ways to build" meeting but if we want to continue using vcpkg, it makes sense to not have openssl under the prereqs script. For now, I will leave it here but it should be trivial to move this to prereqs if we decide to go that way after the meeting.